### ISSUE TITLE
Would a patchlevel release be possible or is planned already ?

### ISSUE THREAD

**Issue by sbuerk** [firebase/php-jwt#578]
Is there any plan for a next (patchlevel) release ? Could not ind any release schedule.

Specially regarding the PHP8.4 required changes of merged #572 ?

**Comment by lolli42** [comment]
ping.

**Comment by lolli42** [comment]
Ping.
PHP is at 8.4.0 rc2, would be great to have a compat release of this library.

**Comment by corrosive4354** [comment]
You should often use different versions of PHP.
In fact, you can use the [Servbay](servbay.com) tool, which supports PHP8.4beta, without worrying about environmental pollution.

You can switch at any time

**Comment by ruudk** [comment]
@bshaffer You are probably very busy, but could you please tag a new release so that projects that rely on this have a few less errors to worry about when preparing for PHP 8.4 😅 🙏

**Comment by ruudk** [comment]
@ajupazhamayil would you be able to help out with this?

**Comment by standa** [comment]
If this helps to anyone: we are running on the commit pin (which seems to work for the time being).

```json
	"require": {
		"firebase/php-jwt": "dev-main#76808fa",
}
```

Just note `composer validate` complains a bit 😞

```
# General warnings
- The package "firebase/php-jwt" is pointing to a commit-ref, this is bad practice and can cause unforeseen issues.
```

**Comment by sbuerk** [comment]
Since PHP 8.4.1 has been released some days ago, could we expect a `patchlevel` release of this library and merged fixes after these couple of months?

It does not show much trust if a framework needs to say:

> You can use our framework safely with PHP8.4, but as we rely on XY you need to use a composer patch for already merged but not released PHP8.4 fixes.

to announce official PHP8.4 support.

**Comment by hafezdivandari** [comment]
It seems that the repo is using the [release-please bot](https://github.com/firebase/php-jwt/commits?author=release-please%5Bbot%5D) for releases, and this bot [doesn't take `chore` commits into account](https://github.com/googleapis/release-please#:~:text=A%20%22chore%22%20or%20%22build%22%20commit%20is%20not%20a%20releasable%20unit.)!

**Comment by sbuerk** [comment]
> It seems that the repo is using the [release-please bot](https://github.com/firebase/php-jwt/commits?author=release-please%5Bbot%5D) for releases, and this bot [doesn't take `chore` commits into account](https://github.com/googleapis/release-please#:~:text=A%20%22chore%22%20or%20%22build%22%20commit%20is%20not%20a%20releasable%20unit.)!

Damn. Should have made one of the pr commits a `fix`. What a pity that is not mentioned within the README.md.

**Comment by ruudk** [comment]
So would it help if we do a new PR with a proper commit to get it tagged? There's probably something tiny code style thing we can tweak / fix.

**Comment by ruudk** [comment]
@hafezdivandari @sbuerk I think it would be solved with https://github.com/firebase/php-jwt/pull/583, agreed?

**Comment by sbuerk** [comment]
> @hafezdivandari @sbuerk I think it would be solved with #583, agreed?

not sure if `deps` would lead to a release (don't know release-please) ... not that we have the same issue with the pr because it was only `chore`.

**Comment by hafezdivandari** [comment]
They usually squash before merge, so the `fix` prefix on the #583 PR title makes it a releasable unit 👍

**Comment by ruudk** [comment]
This can be closed

https://github.com/firebase/php-jwt/pull/585
